File: project/backend/models/exercisequestion.js
"use strict";
module.exports = (sequelize, DataTypes) => {
  const ExerciseQuestion = sequelize.define("ExerciseQuestion", {
    question_id: {
      allowNull: false,
      autoIncrement: true,
      primaryKey: true,
      type: DataTypes.INTEGER
    },
    lang_abbr: {
      type: DataTypes.STRING
    },
    exercise_id: {
      type: DataTypes.INTEGER
    },
    answer_id: {
      type: DataTypes.INTEGER
    },
    desc: {
      type: DataTypes.STRING
    },
    media_url: {
      type: DataTypes.STRING
    },
    media_type: {
      type: DataTypes.STRING
    },
    media_start_time: {
      type: DataTypes.INTEGER
    },
    media_end_time: {
      type: DataTypes.INTEGER
    }
  });
  ExerciseQuestion.associate = function(models) {
    models.ExerciseQuestion.hasMany(models.ExerciseChoice, {
      foreignKey: "question_id",
      as: "choices",
      constraints: false
    });
    models.ExerciseQuestion.belongsTo(models.ExerciseChoice, {
      foreignKey: "answer_id",
      as: "answer",
      constraints: false
    });
  };
  return ExerciseQuestion;
};

File: project/mobile/bulingo/app/src/main/java/com/bulingo/Database/ExerciseQuestion.java
package com.bulingo.Database;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ExerciseQuestion {

    @SerializedName("question_id")
    public String id;

    @SerializedName("desc")
    public String desc;

    @SerializedName("media_url")
    public String media_url;

    @SerializedName("media_start_time")
    public String media_start_type;

    @SerializedName("media_end_time")
    public String media_end_time;

    @SerializedName("choices")
    public List<Choice> choices;

    public class Choice {
        @SerializedName("id")
        public int id;
        @SerializedName("desc")
        public String desc;
    }


}
